pkg/service/rpc/io.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package rpc;
+
+option go_package = "github.com/livekit/livekit/pkg/service/rpc";
+
+import "livekit_rpc_internal.proto";
+
+service IOInfo {
+  rpc GetIngressInfo(livekit.GetIngressInfoRequest) returns (livekit.GetIngressInfoResponse);
```

Comment:
let's add UpdateEgressInfo as well

Reply:
moving to a separate pr per slack discussion